anagram.cpp

```diff
@@ -1,13 +1,39 @@
 #include "anagram.h"
+#include <cctype>
+#include <algorithm>
 
-bool Anagram::WordPairIsAnagram(const std::string& word1, const std::string& word2) {
-    //Fill the correct implementation here
+bool Anagram::WordPairIsAnagram(const std::string& word1, const std::string& word2)
+{
+	auto word_one = word1;
```

Comment:
Clear implementation 👍 
The erase, toupper and sort are happening twice in this function. Can you make a pre-processing function that can do all of those?
Add tests to take care of the first few examples in Wikipedia. You'll see that there is a fix needed, which you can do in the single function itself.